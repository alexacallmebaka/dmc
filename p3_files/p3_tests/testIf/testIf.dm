main  :   () void {

if( (-1 + 4) == -(2 + 3) ) { a : int = (47/8)*6; }
if(!(3>=(-7)) ) {}
if( !((!4 and 6) or ( 5 and 7)) ) {}
}
