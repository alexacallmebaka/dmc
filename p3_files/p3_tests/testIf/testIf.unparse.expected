main : () void  {
    if (1){
        a : int = 0;
    }
    if (4){
    }
}
